open_event/static/js/jquery/jquery.codezero.js

```diff
@@ -0,0 +1,221 @@
+/**
+ * Created by Niranjan on 29-May-16.
+ */
+
+/**
+ * An extended jQuery method to call a callback on an input change
+ */
+jQuery.fn.extend({
+    valueChange: function (callback) {
+        return this.each(function () {
+            var elem = $(this);
+            elem.data('oldVal', elem.val());
+            elem.bind("propertychange change click keyup input paste", function (event) {
+                if (elem.data('oldVal') != elem.val()) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '!==' and instead saw '!='.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2685681420/issues/source?bid=3233864&fileBranchId=3324174#l14)